testsuite/features/profiles/Docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.mgr.suse.de/toaster-sles15sp2-products
+FROM registry.suse.com/suse/sle15:15.4
```

Comment:
No, we don't want to rely on that registy.
We want to use FROM registry.mgr.suse.de/suse/sle15:15.4

Reply:
Damn, I copy pasted the wrong URL. Thank you Oscar! This was not intentional.

Reply:
Fixed.